### PR TITLE
Add content to work experience section and make formatting and style changes

### PR DESCRIPTION
**Pull request by kyracq**
Preview: https://htmlpreview.github.io/?https://github.com/kyracq/step/blob/development/portfolio/src/main/webapp/index.html

### CODE REVIEW

**Review by otobin — APPROVED** [review]
Looks awesome Kyra! I really like the carousel of work experience and the style on the first page. I think some of your indentations are 4 instead of two, but the rest of the code is super clear and very nicely commented. The icon on the tab with your initials is also a really nice touch!

**Review by shulijones — APPROVED** [review]
LGTM! I can tell you made some changes to bring it even more in line with the style guide. I really like the jobs carousel along the bottom of the homepage as well.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,17 +12,77 @@
...
 // limitations under the License.
 
+var index = 0;
+const jobCount = 3;
+
+/**
+ * Set innerHTML text in work experience section
+ */
+function setContent() {
+  const jobs = ['UI Development Intern @ Accenture', 'STEP Intern @ Google', 'STEP Intern @ Google'];
```
Comment (nsri-google) [inline portfolio/src/main/webapp/script.js:22]: you might want to consider an array of objects. something that looks like

jobs = [
{name: "UI development...", dates: "Sept, 2017 - ...", summary: "Used reactjs..."},
{name: "STEP...",...},
{...},
;]

Comment (nsri-google) [reply portfolio/src/main/webapp/script.js:22]: This way you can group information together and depend on them being at the same index in each array.

you can refer to this as jobs[index].name jobs[index].dates and jobs[index].summary

**Review by nsri-google — APPROVED** [review]

**Pull request merged**